common/persistence/src/test/java/app/coronawarn/server/common/persistence/domain/DiagnosisKeyBuilderTest.java

```diff
@@ -218,12 +247,20 @@ private DiagnosisKey keyWithRiskLevel(int expTransmissionRiskLevel) {
         .withTransmissionRiskLevel(expTransmissionRiskLevel).build();
   }
 
+  private DiagnosisKey keyWithSubmissionTimestamp(long submissionTimestamp) {
```

Comment:
You could re-use ``DiagnosisKeyServiceTest.buildDiagnosisKeyForSubmissionTimestamp`` instead. Even though it might be placed a little awkwardly there, I assume we can extract it to somewhere else in the future.

Reply:
thanks, I hadn't noticed that. I extracted and reused the methods from the package private test class to the already existing public helper class now.